benchmarks/config/base/base.json

```diff
@@ -113,4 +113,31 @@
       "extensions": [ "avx2" ]
     }
   }}
+  {
+    "pack": {
+      "gemm_operand_a_512x1024": {
```

Comment:
Call them by their types `fp32` / `bf16`, since soon we'll have both types of tests.